### PR TITLE
ARR - Adjusting review feedback from

### PR DESCRIPTION
**Pull request by jkkummerfeld**
The ARR EICs have designed a new review feedback from (effort led by @annargrs). I've updated the form itself and the description of the date fields in the venue configuration (so it is clear to future EICs what those dates are for).

Is there any chance this can be applied to the August 2024 ARR venue? The idea would be to make the form accessible to authors in late October. I completely understand if the answer is no, since the venue is live already, but wanted to check.

Thanks!

### CODE REVIEW

**Comment by haroldrubio** [comment]
Yes, we won't have time to deploy this for August - I'll modify the tests to reflect the new workflow.

The tests are currently configured to have this open up after meta-reviews are released but would it be better for this to be open during the meta-reviewing stage?

**Comment by jkkummerfeld** [comment]
This should be released during the author response period.

**Comment by jkkummerfeld** [comment]
@haroldrubio Any status update here? We'd like to get it merged in before the October site is set up. Thanks!

**Comment by jkkummerfeld** [comment]
Also, what will the title of this be? Just 'Official Comment'? Or something else? Ideally, it would be different, so it is easy to identify.

**Comment by haroldrubio** [comment]
I've moved around the existing tests - you can see a preview of the form below. The form is called "Review Rating" by default
Screenshot 2024-09-18 at 9 34 29 AM

Is the wording finalized? I'll need to update the tests another time and I can ask for reviews on the changes after that.

**Comment by jkkummerfeld** [comment]
Thanks, yes, the wording is finalised.

One thing to check (that is not clear from the screenshot) - this should be a form that is optionally completed once per review. Is it clear which review it is attached to / about?

**Comment by annargrs** [comment]
Thanks so much for the screenshot, Harold! Would it be possible to rename that form to "Review Issue Report"? As you can see, there is no field there that can be interpreted as a rating, so this title might be confusing.

**Comment by jkkummerfeld** [comment]
Two things:

1 - I was trying to make further edits and may have messed up merging etc. Sorry! Hopefully easy to resolve

2 - The thing I was adding was changes to the metareview form that go with this PR. One thing we don't have that is needed is limiting who can see certain fields in the metareview. I can see [here](https://docs.openreview.net/how-to-guides/submissions-comments-reviews-and-decisions/how-to-hide-reveal-fields) how to do that for a single venue, but how can I do it generically? To be specific, we want the following fields in the metareview to have the following visibility:

Authors, AC, SAC, PC
- reported_issues
- note_to_authors

AC, SAC, PC
- best_paper_ae
- best_paper_ae_justification
- ethical_concerns
- needs_ethics_review
- author_identity_guess
- great_reviews
- poor_reviews
- explanation

**Comment by annargrs** [comment]
@haroldrubio could you confirm when this would be merged? I'm an EiC for this cycle, and we need to publish the new version of reviewer instructions asap, and the instructions assume that this new process is in place.

**Comment by haroldrubio** [comment]
Hi @annargrs we'll be able to get this ready for this cycle, I'll ask for another review and hopefully we could get this merged next week

File: openreview/arr/helpers.py [hunk 1]
```diff
@@ -260,14 +260,14 @@ class ARRWorkflow(object):
             "order": 37,
             "required": False
         },
-        "review_rating_start_date": {
-            "description": "When should the review rating form open?",
+        "review_issue_start_date": {
+            "description": "When should the form for authors to make sturctured complaints to ACs about reviews open?",
```
Comment (celestemartinez) [inline openreview/arr/helpers.py:264]:
```suggestion
            "description": "When should the form for authors to make structured complaints to ACs about reviews open?",
```

File: openreview/arr/helpers.py [hunk 2]
```diff
@@ -260,14 +260,14 @@ class ARRWorkflow(object):
...
             "value-regex": "^[0-9]{4}\\/([1-9]|0[1-9]|1[0-2])\\/([1-9]|0[1-9]|[1-2][0-9]|3[0-1])(\\s+)?((2[0-3]|[01][0-9]|[0-9]):[0-5][0-9])?(\\s+)?$",
             "order": 38,
             "required": False
         },
-        "review_rating_exp_date": {
-            "description": "When should the review rating form close?",
+        "review_issue_exp_date": {
+            "description": "When should the form for authors to make sturctured complaints to ACs about reviews close?",
```
Comment (celestemartinez) [inline openreview/arr/helpers.py:270]:
```suggestion
            "description": "When should the form for authors to make structured complaints to ACs about reviews close?",
```

File: openreview/stages/arr_content.py [hunk 3]
```diff
@@ -2501,277 +2538,187 @@
...
-                        "description": "4 = (Above Average) - Main contributions and shortcomings of the work are correctly identified and the work is assessed fairly. Nonetheless, the review still contains some (minor) incorrect observations or unsupported claims."
-                    },
-                    {
-                        "value": 5,
-                        "description": "5 = (Solid) - The review is thorough and accurate. The reviewer has correctly identified and fairly judged all strengths and shortcomings of the work. Claims are substantiated and suggestions and/or criticism constructive."
-                    }
+                    "The review is not specific enough."
                 ],
-                "type": "integer"
+                "optional": True,
```
Comment (celestemartinez) [inline openreview/stages/arr_content.py:2548]: should this checkbox be `"deletable": True` as well?

**Review by celestemartinez — APPROVED** [review]

**Pull request merged**